doc/source/tutorial/neighborfinding.rst

```diff
@@ -0,0 +1,217 @@
+.. _neighbors:
+
+=================
+Neighbor Finding
+=================
+
+Problem Statement
+=================
+
+A central task in many of the computations in **freud** is finding nearest neighbors.
```

Comment:
`finding particles' neighbors.` (so as not to confuse this statement with _k_-nearest neighbors queries)